### DISCUSSION TITLE
Add configuration (monorepo/group preset) for nuget packages Microsoft.Identity.Client[.*]

### DISCUSSION THREAD

**Discussion by Styxxy** [renovatebot/renovate#32768] [Suggest an Idea]
### Tell us more.

I noticed the repo nuget libraries `Microsoft.Identity.Client[.*]` are missing in any of the known group./monorepo presets.
Is it possible to add this as a preset?

Repo: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet
Nuget packages:
- [Microsoft.Identity.Client.Broker](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker)
- [Microsoft.Identity.Client.Desktop](https://www.nuget.org/packages/Microsoft.Identity.Client.Desktop)
- [Microsoft.Identity.Client.Extensions.Msal](https://www.nuget.org/packages/Microsoft.Identity.Client.Extensions.Msal)
- [Microsoft.Identity.Client](https://www.nuget.org/packages/Microsoft.Identity.Client)

**Comment by rarkins** [comment]
Yes, PR welcome
